src/actions/channel_categories.ts

```diff
@@ -234,6 +234,85 @@ export function moveChannelToCategory(categoryId: string, channelId: string, new
     };
 }
 
+export function moveChannelsToCategory(categoryId: string, channelIds: string[], newIndex: number, setManualSorting = true) {
```

Comment:
You probably don't need the `setManualSorting` parameter with this one. `moveChannelToCategory` singular only has that for `addChannelToCategory`.

Reply:
We'll need this for favouriting and muting as a bulk operation.